pywis_pubsub/publish.py

```diff
@@ -164,7 +167,15 @@ def create_message(topic: str, content_type: str, url: str, identifier: str,
             }]
     }
 
-    if operation != 'canonical':
+    if None not in [start_datetime, end_datetime]:
+        LOGGER.debug('Setting time extent')
+        message['properties']['start_datetime'] = start_datetime
+        message['properties']['endstart_datetime'] = end_datetime
+    else:
+        LOGGER.debug('Setting time instant')
+        message['properties']['datetime'] = datetime_
```

Comment:
When not specifying datetime, this will result in a message with properties['datetime'] = None , is that ok/allowed by the standard ?

Reply:
Yes, per Requirement 9A.